samples/apps/copilot-chat-app/webapi/Skills/OpenApiSkills/KlarnaSkill/openapi.json

```diff
@@ -0,0 +1,120 @@
+{
+  "openapi": "3.0.1",
+  "info": {
+    "version": "v0",
+    "title": "Open AI Klarna product Api"
+  },
+  "servers": [
+    {
+      "url": "https://www.klarna.com/us/shopping"
```

Comment:
nit: potentially not an issue or a concern. 
But it might make sense to have the user pass in a URL or country name for klarna from the options on https://www.klarna.com/ instead of limiting it to the us which might be slow or blocked for a small number of users.